server/lib/poller.js

```diff
@@ -63,31 +66,62 @@ class Poller
         } catch (err) {
             log.error(`Error when updating Weboob in polling: ${err.message}`);
         }
-
+        let checkAccounts = false;
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            checkAccounts = await Config.findOrCreateDefaultBooleanValue(
                 'weboob-auto-merge-accounts'
             );
+        } catch (err) {
+            log.error(`Could not retrieve 'weboob-auto-merge-accounts':
+                ${err.toString()}`);
+        }
 
-            log.info('Checking new operations for all accesses...');
-            if (checkAccounts) {
-                log.info('\t(will also check for accounts to merge)');
-            }
-
+        // We go on even if the parameter weboob-auto-merge-accounts is
+        // not caught. By default, the merge is not done.
+        log.info('Checking new operations for all accesses...');
+        if (checkAccounts) {
+            log.info('\t(will also check for accounts to merge)');
+        }
+        try {
             let accesses = await Access.all();
             for (let access of accesses) {
                 let accountManager = new AccountManager;
-                if (checkAccounts) {
-                    await accountManager.retrieveAccountsByAccess(
-                        access,
-                        false
-                    );
+                try {
+                    // Only import if last poll did not raise a
+                    // login/parameter error
+                    if (access.canAccessBePolled()) {
+                        if (checkAccounts) {
+                            await accountManager.retrieveAccountsByAccess(
+                                access, false);
+                        }
+                        await accountManager.retrieveOperationsByAccess(
+                            access, cb);
+                    } else {
+                        let error = 'Cannot poll, last import raised:';
+                        log.info(`${error} ${access.fetchStatus}`);
+                    }
+                } catch (err) {
+                    log.error(`Error when polling accounts:
+                        ${err.message}`);
+                    if (err.errCode) {
+                        if (err.errCode ===
+                            getErrorCode('NO_PASSWORD') &&
+                            !this.sentNoPasswordNotification) {
+                            // TODO do something with this
+                            this.sentNoPasswordNotification = true;
```

Comment:
Then the condition above can become `if (err.errCode && isCredentialError(err))`